webpack/assets/javascripts/react_app/common/urlHelpers.js

```diff
@@ -39,7 +39,7 @@ export const getURIsearch = () => getURI().query(true).search || '';
  */
 export const getParams = () => ({
   page: getURIpage(),
-  perPage: getURIperPage(),
+  perPage: getURIperPage() || 25,
```

Comment:
why is that here? foreman hard coded default is actually 20, wont you get this as part of the reactapp?